src/resources/js/docs.js

```diff
@@ -29,59 +37,120 @@ $(function() {
 	// the <pre>; this line finds those and adds a .chroma class
 	// to the outer pre element, and our CSS file has a style to
 	// ensure the inner code block does not produce extra padding
-	$('article > pre:not(.chroma) > code:not(.cmd)').parent().addClass('chroma');
+	$("article > pre:not(.chroma) > code:not(.cmd)")
+		.parent()
+		.addClass("chroma");
 
 	// Add links to Caddyfile directive tokens in code blocks.
 	// See include/docs-head.html for the whitelist bootstrapping logic
-	$('pre.chroma .k')
-		.filter((k, item) =>
-			window.CaddyfileDirectives.includes(item.innerText)
-				|| item.innerText === '<directives...>'
+	$("pre.chroma .k")
+		.filter(
+			(k, item) =>
+				window.CaddyfileDirectives.includes(item.innerText) ||
+				item.innerText === "<directives...>"
 		)
-		.map(function(k, item) {
+		.map(function (k, item) {
 			let text = item.innerText;
-			let url = text === '<directives...>'
-				? '/docs/caddyfile/directives'
-				: '/docs/caddyfile/directives/' + text;
-			text = text.replace(/</g,'&lt;').replace(/>/g,'&gt;');
-			$(item).html('<a href="' + url + '" style="color: inherit;" title="Directive">' + text + '</a>');
+			let url =
+				text === "<directives...>"
+					? "/docs/caddyfile/directives"
+					: "/docs/caddyfile/directives/" + text;
+			text = text.replace(/</g, "&lt;").replace(/>/g, "&gt;");
+			$(item).html(
+				'<a href="' +
+					url +
+					'" style="color: inherit;" title="Directive">' +
+					text +
+					"</a>"
+			);
 		});
 
 	// Add links to [<matcher>] or named matcher tokens in code blocks.
 	// The matcher text includes <> characters which are parsed as HTML,
 	// so we must use text() to change the link text.
-	$('pre.chroma .nd')
-		.map(function(k, item) {
-			let text = item.innerText.replace(/</g,'&lt;').replace(/>/g,'&gt;');
-			$(item).html('<a href="/docs/caddyfile/matchers#syntax" style="color: inherit;" title="Matcher token">' + text + '</a>');
-		});
+	$("pre.chroma .nd").map(function (k, item) {
+		let text = item.innerText.replace(/</g, "&lt;").replace(/>/g, "&gt;");
+		$(item).html(
+			'<a href="/docs/caddyfile/matchers#syntax" style="color: inherit;" title="Matcher token">' +
+				text +
+				"</a>"
+		);
+	});
+
+	// select all bash code elements and dynamically add clipboard button/svg
+	$("pre > code.bash").map(function (_, block) {
```

Comment:
Ah, you only added it to `code.bash`, I see. I think in some cases it's `code.cmd > span.bash` as well. Or something like that.

Reply:
Good catch! I've reduced the specificity of the selector and added span.bash.